src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -10,13 +10,24 @@
   configure and enable the monitoring functionality.</cd-alert-panel>
 
 <cd-alert-panel type="info"
-                *ngIf="!dashboardExist"
-                i18n>Grafana Dashboard doesn't exist. Please refer to
-  <a href="{{ docsUrl }}"
-     target="_blank">documentation</a> on how to
-  add dashboards to Grafana.</cd-alert-panel>
+                *ngIf="grafanaExist && !dashboardValidated"
+                i18n>The requested Grafana Dashboard could not be validated. Possible reasons for this are:
```

Comment:
Hmm, "validated" sound wrong to me in this context. How about "displayed"?
```suggestion
                i18n>The requested Grafana Dashboard could not be displayed. Possible reasons for this are:
```